staff/demi/task-app-complete/task-app/src/index.js

```diff
@@ -0,0 +1,64 @@
+require('dotenv').config()
+
+const express = require('express')
+
+const bodyParser = require('body-parser')
+
+const taskApi = require('./taskapi')
+
+const app = express()
+
+
+app.set('view engine', 'pug')
+
+app.get('/', (req, res) => {
+    
+    dones = todos = []
+
+    taskApi.getTasks().then(tasks => {
+
+        const todos = tasks.filter(task => task.status == false)
```

Comment:
Nos parece curioso que al declarar las variables `dones` y `todos` en la linea 16 no usais `let`, `const` o `var`. Sin embargo si lo hacéis en la línea 20 y 22.